memoization/scripts/main_fibonacci.js

```diff
@@ -0,0 +1,54 @@
+// CHALLENGE --> Efectuar una función, que admita una posición (numero), y devuelva el numero fibonacci
+// Memoization
+
+// function fibonacci(10) // Return, el numero fibonacci --> 34
+//la sucesion empieza con 0 y 1
+
+// «cada término es la suma de los dos anteriores»
+// 0, 1,1 2
+
+let fibonacci = number => {
+    let fiboArr = [];
+    let initialNumber0 = 0;
+    for(let i = 0; i<number; i++){
+        let result;
+        i < 2 ? (result = 0 + initialNumber0, initialNumber0++) : result = fiboArr[fiboArr.length-1] + fiboArr[fiboArr.length-2]
+        fiboArr.push(result)
+    }
+    return fiboArr.pop()  
+}
+console.log(fibonacci(10))
+
+///
+const fibonnacci = (number) => {
+    let cache = {};
+    let fiboArr = [];
+    let initialNumber0 = 0;
+
+    return (n) => {
+        if (n in cache) {
```

Comment:
Hey @cdemiguel All your features run right! but as I tried to tell you, if you want to see better the `memoization` you should try to do it with recursive function. But all okey, that an other way to do it!  good job 👍